### PR TITLE
PTVSD debugger

### PR DESCRIPTION
**Pull request by Swamp-Ig**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23336

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/ptvsd.markdown [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+title: "PTVSD Debugger"
+description: "Debugging from visual studio code."
+date: 2019-04-24 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: "Utility"
+redirect_from: /components/ptvsd/
```
Comment (klaasnicolaas) [inline source/_components/ptvsd.markdown]: You can remove this, only for already existing pages.

File: source/_components/ptvsd.markdown [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+layout: page
+title: "PTVSD Debugger"
+description: "Debugging from visual studio code."
+date: 2019-04-24 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: "Utility"
```
Comment (klaasnicolaas) [inline source/_components/ptvsd.markdown]:
```suggestion
ha_category:
  - Utility
```

File: source/_components/ptvsd.markdown [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+redirect_from: /components/ptvsd/
+ha_qa_scale: internal
+ha_release: 0.92
```
Comment (klaasnicolaas) [inline source/_components/ptvsd.markdown]:
```suggestion
ha_release: 0.93
```

File: source/_components/ptvsd.markdown [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+---
+
+The `ptvsd` component allows you to connect the ptvsd debugger from visual studio code to connect to home assistant.
+
+To enable the `ptvsd` component add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
```
Comment (klaasnicolaas) [inline source/_components/ptvsd.markdown:23]: Maybe better if you place the other examples under the configuration variables section.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/ptvsd.markdown [hunk 5]
```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: "PTVSD Debugger"
+description: "Debugging from visual studio code."
+date: 2019-04-24 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
```
Comment (balloob) [inline source/_components/ptvsd.markdown]: Let's drop the HA logo. Let's just use no logo.

File: source/_components/ptvsd.markdown [hunk 6]
```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: "PTVSD Debugger"
+description: "Debugging from visual studio code."
```
Comment (balloob) [inline source/_components/ptvsd.markdown]:
```suggestion
description: "Debugging from Visual Studio Code."
```

File: source/_components/ptvsd.markdown [hunk 7]
```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: "PTVSD Debugger"
```
Comment (balloob) [inline source/_components/ptvsd.markdown]: I suggest we add `(Visual Studio Code)` behind the name, so people know what it's about.

```suggestion
title: "PTVSD Debugger (Visual Studio Code)"
```

**Comment by Swamp-Ig** [comment]
All suggested changes accepted.

File: source/_components/ptvsd.markdown [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+title: "PTVSD Debugger (Visual Studio Code)"
+description: "Debugging from Visual Studio Code."
+date: 2019-04-24 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: 
+  - Utility
+ha_qa_scale: internal
```
Comment (klaasnicolaas) [inline source/_components/ptvsd.markdown]: This is not correct

File: source/_components/ptvsd.markdown [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+  type: string
+port:
+  description: Port to listen on.
+  required: false
+  default: 5678
+  type: port
+wait:
+  description: If true, wait for the debugger to connect before starting up home assistatn.
+  required: false
+  default: False
```
Comment (klaasnicolaas) [inline source/_components/ptvsd.markdown]:
```suggestion
  default: false
```

File: source/_components/ptvsd.markdown [hunk 10]
```diff
@@ -0,0 +1,118 @@
+---
```
Comment (klaasnicolaas) [inline source/_components/ptvsd.markdown:1]: perhaps use the Visual Studio Code logo?

File: source/_components/ptvsd.markdown [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+comments: false
+sharing: true
+footer: true
+ha_category: 
+  - Utility
+ha_qa_scale: internal
+ha_release: 0.93
+---
+
+The `ptvsd` component allows you to connect the ptvsd debugger from visual studio code to connect to home assistant.
```
Comment (klaasnicolaas) [inline source/_components/ptvsd.markdown]: perhaps some extra info what you can do with this component, what can you use it for etc.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/ptvsd.markdown (hunk 10 above)
Comment (Swamp-Ig) [reply source/_components/ptvsd.markdown:1]: Done

**Comment by Swamp-Ig** [comment]
Changes pushed as requested.

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**